CHANGELOG.md

```diff
@@ -32,6 +32,8 @@ If possible, provide tooling that performs the changes, e.g. a shell-script.
 
 * Added `seqan3::phred94`, a quality type that represents the full Phred Score range (Sanger format) and is used for
   PacBio Phred scores of HiFi reads ([\#2290](https://github.com/seqan/seqan3/pull/2290)).
+* Deprecated `seqan3::phred68legacy`. Use `seqan3::phred68solexa` instead.
+  ([\#...](https://github.com/seqan/seqan3/pull/2522)).
```

Comment:
```suggestion
  ([\#2522](https://github.com/seqan/seqan3/pull/2522)).
```

Reply:
I would rather write `renamed X to Y`
Deprecation is usually more for stuff that does not exist anymore